test/libsolidity/smtCheckerTests/literalSuffixes/equivalent_function_call_with_two_arguments.sol

```diff
@@ -0,0 +1,17 @@
+function suffix(uint m, uint e) pure suffix returns (uint) {
+    return m + e;
+}
+
+contract C {
+    function test() public pure {
+        assert(1.23 suffix == suffix(123, 2));
+    }
+}
+
+// ====
+// SMTEngine: all
+// ----
+// Warning 6031: (144-155): Internal error: Expression undefined for SMT solver.
+// Warning 7520: (144-155): Literal suffix functions with two arguments are not yet supported by SMTChecker. The value of the literal will not be affected by the suffix, which may lead to incorrect results.
+// Warning 6328: (137-174): CHC: Assertion violation happens here.
+// Info 1391: CHC: 1 verification condition(s) proved safe! Enable the model checker option "show proved safe" to see all of them.
```

Comment:
Wait, what verification condition was proved safe here? The assertion fails above, so this would probably be the overflow check in `m + e`, but that should fail, shouldn't it?

Reply:
Hum, not sure, but I am guessing that it doesn't fail because `123 + 2` is not overflow (?)

Reply:
Yea but that's just one case. There's also the general case `m + e` inside the function.

Reply:
Ahh function `suffix` is internal though, right?

Reply:
Yes, suffixes are always internal calls.

Reply:
Ok, then yea that property is proved safe because it's only reachable with `123 + 2`.